components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -63,9 +63,11 @@ impl Manifest {
     /// [`clap::ArgMatches`]: https://kbknapp.github.io/clap-rs/clap/struct.ArgMatches.html
     pub fn new_from_cli_matches(_ui: &mut UI, matches: &ArgMatches) -> Result<Self> {
         let count = matches.value_of("COUNT").unwrap_or("1").parse()?;
-        let topology: Topology = FromStr::from_str(
-            matches.value_of("TOPOLOGY").unwrap_or("standalone"),
-        ).unwrap_or(Default::default());
+        let topology: Topology = matches
+            .value_of("TOPOLOGY")
+            .unwrap_or("standalone")
+            .parse()
+            .unwrap_or(Default::default());
```

Comment:
How about using `Topology::default()`? Would it spare you from explicitly specifying the Topology type next to the variable name?

Reply:
I'm not sure. Even the usage of topology later might be enough to tell rustc which type to pick. I'll check..

Reply:
I don't see the point of this change, since now we are using the default two times, which is ugly. I think it looked better before, with maybe `String::from` instread of `FromStr::from_str`.

Reply:
Are we? I only see 1 call to Default::default() here. If you mean the both components of the call then it was the same previously as well. :) The change here is not the Default::default but rather Fromstr::from_str() -> parse(), which is what I was explicitly asked to do in the Helm exporter PR by cm and is the general convention in Rust world afaik. If you had reviewed individual commits instead, it'd have been very obvious. :)

Reply:
I did review individual commits. The point about two defaults is the `unwrap_or("standalone")` and `unrwap_or(Default:::default())`.

Reply:
The actual value of the default is quite unimportant here and redundant information here. "Just use the default topology" is what we are saying here.